Open Judge System/Data/OJS.Data/Migrations/DefaultMigrationConfiguration.cs

```diff
@@ -313,6 +314,36 @@ protected void SeedSubmissionTypes(OjsDbContext context)
                     AllowedFileExtensions = "zip",
                     AllowBinaryFilesUpload = true,
                 },
+                new SubmissionType
+                {
+                    Name = "C# Unit Tests Tester",
+                    CompilerType = CompilerType.None,
+                    AdditionalCompilerArguments = "--noresult --inprocess",
+                    ExecutionStrategyType = ExecutionStrategyType.CSharpUnitTestsRunnerExecutionStrategy,
```

Comment:
Here we have `"CSharpUnitTestsRunner"`, on row 319 we have name that is `"C# Unit Tests Tester"`. IMO choose `"runner"` or `"tester"` and use in both places